packages/react/src/experimental/RichText/RichTextEditor/index.tsx

```diff
@@ -63,379 +65,411 @@ type RichTextEditorHandle = {
+          setLastIntent={setLastIntent}
+          toolbarLabels={toolbarLabels}
+          setIsToolbarOpen={setIsToolbarOpen}
+          isToolbarOpen={isToolbarOpen}
+        />
 
-          {filesConfig && (
-            <>
-              <input
-                id="rich-text-editor-upload-button"
-                type="file"
-                multiple={filesConfig.multipleFiles}
-                onChange={handleFileChange}
-                ref={fileInputRef}
-                className="hidden"
-                accept={getAcceptFileTypeString(filesConfig)}
-                aria-label="Upload file"
-              />
-              <AnimatePresence>
-                {files.length > 0 && (
-                  <motion.div
-                    key="filelist-accordion"
-                    initial={{ height: 0, opacity: 0, y: -20 }}
-                    animate={{ height: "auto", opacity: 1, y: 0 }}
-                    exit={{ height: 0, opacity: 0, y: -20 }}
-                    transition={{ duration: 0.3 }}
-                  >
-                    <FileList
-                      files={files}
-                      onRemoveFile={(fileIndex) =>
-                        handleRemoveFile(
-                          fileIndex,
-                          files,
-                          filesConfig,
-                          setFiles
-                        )
-                      }
-                      disabled={disableAllButtons}
-                    />
-                  </motion.div>
-                )}
-              </AnimatePresence>
-            </>
-          )}
-          <Footer
-            editor={editor}
-            maxCharacters={maxCharacters}
-            secondaryAction={secondaryAction}
-            primaryAction={primaryAction}
-            fileInputRef={fileInputRef}
-            canUseFiles={filesConfig ? true : false}
-            isLoadingEnhance={isLoadingEnhance}
-            disableButtons={disableAllButtons}
-            enhanceConfig={enhanceConfig}
-            isFullscreen={isFullscreen}
-            onEnhanceWithAI={handleEnhanceWithAI}
-            setLastIntent={setLastIntent}
-            toolbarLabels={toolbarLabels}
-            setIsToolbarOpen={setIsToolbarOpen}
-            isToolbarOpen={isToolbarOpen}
-          />
+        <EditorBubbleMenu
+          editor={editor}
+          disableButtons={disableAllButtons}
+          toolbarLabels={toolbarLabels}
+          isToolbarOpen={isToolbarOpen}
+        />
+      </div>
+    </div>
+  )
+})
 
-          <EditorBubbleMenu
-            editor={editor}
-            disableButtons={disableAllButtons}
-            toolbarLabels={toolbarLabels}
-            isToolbarOpen={isToolbarOpen}
-          />
+interface RichTextEditorSkeletonProps {
+  rows?: number
+}
+
+const RichTextEditorSkeleton = ({ rows = 2 }: RichTextEditorSkeletonProps) => {
+  const widths = Array.from(
+    { length: rows },
+    () => `${Math.floor(Math.random() * 60) + 40}%`
```

Comment:
could you add a prop to set it static? Otherwise it will always fail screenshot tests due to the randomness